table/lib/azure/storage/table/default.rb

```diff
@@ -30,7 +30,7 @@
 module Azure::Storage::Table
   module Default
     # Default REST service (STG) version number
-    STG_VERSION = "2016-05-31"
```

Comment:
Let's not update the Table service. IIRC, from some version, the default payload was changed to json which we don't handle properly.